core/src/main/resources/reference.conf

```diff
@@ -134,6 +134,17 @@ jdbc-journal {
     }
 
     event_tag {
+      tableName = "event_tags"
```

Comment:
Do we have to change the table name? Is that because it's not possible to alter the primary key? Wouldn't it be difficult to keep the old tags when migrating?

I would like to see the migration instructions added to docs/src/main/paradox/migration.md